src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+const Categories = () => (
+  <section className="categories-container">
+    <h2>Under construction</h2>
+  </section>
+);
```

Comment:
- [ ] Following the project requirements, the `Categories` page should display only a button with the text "Check status". Kindly work on that.